packages/dds/merge-tree/src/test/obliterate.concurrent.spec.ts

```diff
@@ -54,6 +54,30 @@ for (const incremental of [true, false]) {
 			MergeTree.options.incrementalUpdate = true;
 		});
 
+		it("obliterate, then insert at the end of the string", () => {
+			const helper = new ReconnectTestHelper();
+
+			helper.insertText("A", 0, "ABCDEFGH");
+			helper.processAllOps();
+			helper.obliterateRange("A", 0, 8);
+			helper.insertText("B", 8, "123");
+			helper.processAllOps();
+
+			helper.logger.validate({ baseText: "123" });
```

Comment:
nit. I've found that tests are easier to read when the initial text is numbers, and clients just insert their own id, as in a test like this its them very easy to tell that 8 in the length, so the whole string is obliterated, and B is the client who will insert and win.

If you like this, you can make updates, but not required.

```suggestion

			helper.insertText("A", 0, "01234567");
			helper.processAllOps();
			helper.obliterateRange("A", 0, 8);
			helper.insertText("B", 8, "BBB");
			helper.processAllOps();
			
			elper.logger.validate({ baseText: "BBB" });
```

Reply:
I thought about doing this, but I went with the letters to match the rest of the tests in the file. If that's not as important, then I don't mind changing the base text to numbers.